### DISCUSSION TITLE
mouse Position Mapping (like Minecraft boxes, or inventory boxes)

### DISCUSSION THREAD

**Discussion by SideOrder** [AntiMicroX/antimicrox#717] [Ideas]
is there a way to implement a way to put the mouse directly in an area and move so many pixels in a direction? like picking out your inventory in Minecraft or any other inventory. so when you press a key on the controller it will automatically be locked in an invisible barrier and whenever you move your stick it will move Eg: 32 pixels to the left if you press left (you would obviously fine-tune it if it's off that pixel amount) it would go to the next item in your inventory. and once you select that item you could it would change the profile to release the barrier and do what you want with it or something in that manner.

**Comment by pktiuk** [comment]
No, there is no such functionality yet.
